suite-native/formatters/src/components/EthereumTokenSymbolFormatter.tsx

```diff
@@ -4,10 +4,10 @@ import { Text, TextProps } from '@suite-native/atoms';
 import { TokenSymbol } from '@suite-common/wallet-types';
 
 type EthereumTokenSymbolFormatterProps = {
-    ethereumSymbol: TokenSymbol;
+    ethereumSymbol?: TokenSymbol;
 } & TextProps;
 
 export const EthereumTokenSymbolFormatter = ({
     ethereumSymbol,
     ...rest
-}: EthereumTokenSymbolFormatterProps) => <Text {...rest}>{ethereumSymbol.toUpperCase()}</Text>;
+}: EthereumTokenSymbolFormatterProps) => <Text {...rest}>{ethereumSymbol?.toUpperCase()}</Text>;
```

Comment:
FYI: tokens should not be uppercased/lowercased https://github.com/trezor/trezor-suite/issues/8037

Reply:
This could be quite issue, we should create follow-up for this because we are doing this everywhere.